src/pages/DesktopPage.tsx

```diff
@@ -83,11 +83,13 @@ const DesktopPage: React.FC = () => {
                     <IonCard
                       style={{
                         position: "absolute",
+                        margin: "0px",
```

Comment:
The card has a margin, that was screwing up on iPads due to different values in relation to current viewport size